test/services/import/_fixtures/import-licence-versions.fixture.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const { randomInteger } = require('../../../support/general.js')
+
+function importLicenceVersionPurpose () {
+  return {
+    abstractionPeriodEndDay: 31,
+    abstractionPeriodEndMonth: 3,
+    abstractionPeriodStartDay: 1,
+    abstractionPeriodStartMonth: 4,
+    annualQuantity: 545520,
+    dailyQuantity: 1500.2,
+    externalId: `${randomInteger(1, 9)}:${randomInteger(10000004, 99999999)}`,
```

Comment:
These would have to be functions otherwise the generate data will be the same for each test unless we put the require in the bfe. 

We could create / follow the pattern of the model helpers and have it export a single func to generate the data. The issue with this only the effort to desctruce the data from the array formats in this case. 

Often we only want the one purpose in `licenceVersion.purposes[0]`